symengine/symengine_exception.h

```diff
@@ -0,0 +1,34 @@
+#ifndef SYMENGINE_EXCEPTION_H
+#define SYMENGINE_EXCEPTION_H
+
+enum symengine_exceptions_t {
+    NO_EXCEPTION,
+    DIV_BY_ZERO,
+    NOT_IMPLEMENTED,
+    UNDEFINED
+};
+
```

Comment:
The `enum` is defined here and this file is included in `cwrapper.cpp`, right? The C API user includes the `cwrapper.h` file. So when the C API returns an error code what will the error code be compared against?